### DISCUSSION TITLE
How can I get a summary of what was updated by `brew upgrade` without scrolling up?

### DISCUSSION THREAD

**Discussion by allanlewis** [Homebrew/discussions#3609] [Everyday usage]
I run `brew upgrade` frequently and like to know what's been upgraded. My typical workflow is to run `brew upgrade && brew cleanup` in one terminal and then get on with something else in another while the upgrade is running. When the upgrade finishes, I like to know what was upgraded, but AFAICT the only way to do this is to scroll up to the section headed `==> Upgrading _n_ outdated packages`.

It would be really great to have `brew upgrade` print a summary of what it did just before it exits. I'm happy to raise a feature request if that's the best way to make this happen but thought I'd ask here first before adding to the issues list.

**Comment by SMillerDev** [comment]
> `brew upgrade && brew cleanup`

`brew upgrade` will actually run cleanup too.

> I'm happy to raise a feature request if that's the best way to make this happen but thought I'd ask here first before adding to the issues list.

Feature requests should be filed as pull requests, but otherwise feel free.
